### PR TITLE
Dependency updates December 5

### PR DESCRIPTION
**Pull request by JayFlexy**
Dependency updates, new docker image required.

### CODE REVIEW

**Review by jimlerza — APPROVED** [review]

File: scripts/verify-terraform-version.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 tf_version=$(terraform --version)
 
-if [[ ${tf_version} != *"1.9.8"* ]]; then
+if [[ ${tf_version} != *"1.10.1"* ]]; then
   echo "Please set your terraform version to 1.9.8 before deploying."
```
Comment (TomElliottFlexion) [inline scripts/verify-terraform-version.sh]:
```suggestion
  echo "Please set your terraform version to 1.10.1 before deploying."
```

Comment (TomElliottFlexion) [reply scripts/verify-terraform-version.sh]: Tiny nit

**Review by TomElliottFlexion — CHANGES_REQUESTED** [review]
Did this end up getting a migration?

**Comment by JayFlexy** [comment]
> Did this end up getting a migration?

Ah, good catch! Doing now

**Review by pixiwyn — CHANGES_REQUESTED** [review]
This isn't building, possible issues with tf/aws provider updates, been debugging this, please hold on merging this till resolved.

**Comment by pixiwyn** [comment]
This update cannot go as is. Posted message about moving onto next week's dependency updates in USTC Project chat. Closing this PR.

**Pull request closed without merging**